tests/commonTemplates_test.go

```diff
@@ -75,6 +78,39 @@ var _ = Describe("Common templates", func() {
 			err := apiClient.Get(ctx, testTemplate.GetKey(), testTemplate.NewResource())
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		It("[test_id:5352]creates only one default variant per OS", func() {
+			liveTemplates := &templatev1.TemplateList{}
+			err := apiClient.List(ctx, liveTemplates,
+				client.InNamespace(strategy.GetTemplatesNamespace()),
+				client.MatchingLabels{
+					"template.kubevirt.io/version": commonTemplates.Version,
+				},
+			)
+			Expect(err).ToNot(HaveOccurred())
+
+			osDefaultCounts := make(map[string]int)
+			for _, liveTemplate := range liveTemplates.Items {
+				_, isDefaultOSVariant := liveTemplate.Labels["template.kubevirt.io/default-os-variant"]
+
+				for labelKey := range liveTemplate.Labels {
+					if strings.HasPrefix(labelKey, "os.template.kubevirt.io/") {
+						if isDefaultOSVariant {
+							osDefaultCounts[labelKey]++
+							continue
+						}
+						if _, knownOSVariant := osDefaultCounts[labelKey]; !knownOSVariant {
```

Comment:
Map values are already zero-valued, so this block can be dropped (unless I am missing something)

Reply:
In theory yes, here specifically this line is required though.
In the loop bellow where the counts get checked, it iterates over the map. If this line is missing it would never add OSs that do not have a default so the test would not fail if there is a an OS without a default variant.
I can clarify this in a comment as it's easy to overlook.